.github/workflows/pnnl_push_mirror.yml

```
@@ -1,18 +1,22 @@
 name: PNNL Push Mirror
+#test
 
 on: [pull_request]
 
 jobs:
   to_gitlab:
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v1
-      - uses: spyoungtech/mirror-action@master
+      - uses: actions/checkout@v4
+        with:
+          # ref: ${{ github.head_ref }}
+          fetch-depth: 0
+      - uses: yesolutions/mirror-action@master
         with:
           REMOTE: ${{ secrets.GITLAB_REPO_URL }}
           GIT_USERNAME: ${{ secrets.GITLAB_USER }}
           GIT_PASSWORD: ${{ secrets.GITLAB_ACCESS_TOKEN }}
-          GIT_PUSH_ARGS: --push-option=ci.skip --tags --force --prune
+          GIT_PUSH_ARGS: --push-option=ci.skip --tags --force --prune --verbose
           PUSH_ALL_REFS: "true"
```

I think ignoring the HEAD ref will fix this issue.

```suggestion
          PUSH_ALL_REFS: "false"
          GIT_REF: "refs/remotes/origin/*:refs/heads/* ^refs/heads/HEAD"
```

Tuesday we noticed that the version of git had changed from [2.47 on working jobs](https://github.com/eagles-project/mam4xx/actions/runs/12919147873/job/36029141692#step:3:7) to[ 2.48 on the broken jobs](https://github.com/eagles-project/mam4xx/actions/runs/12931925766/job/36066984595#step:3:7). This was due to a [new release](https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20250120.2) of the `ubuntu-22.04` runner image. Testing locally I realized the HEAD ref may be the problem and found that [git 2.48 made a change](https://github.com/git/git/blob/v2.48.0/Documentation/RelNotes/2.48.0.txt#L31-L36) related to it's handling. This change resolved the issue when testing locally.

---

🏅